### ISSUE TITLE
ensureVector extension

### ISSUE THREAD

**Issue by jakobbossek** [berndbischl/BBmisc#60]
I would like to extend `ensurevector` in the following sense:
- parameter `cl` should accept multiple classes:
```splus
ensureVector(x, n = 1L, cl = c("A", "B"))
```

- At the moment `ensureVector` does kind of a simplification of the output.
```splus
ensureVector("classif.rpart", n = 1L, cl = "character") # vector
ensureVector(makeLearner("classif.rpart"), n = 1L, cl = "Learner") # list
```
Sometimes we want a list in any case. So my proposal is to add a logical flag `ensure.list` with default `FALSE`.

**Comment by berndbischl** [comment]
i like what you say.

i would suggest  also:

- make n=1 the default. that is the most used case.

do a PR?

**Comment by jakobbossek** [comment]
Jup. PR incoming soon 😉

**Comment by jakobbossek** [comment]
Done via merged PR #63.

**Comment by jakobbossek** [comment]
Thanks for merging.
